# SBG Ellipse Messages
Header header

# Time since sensor is powered up us 
uint32 time_stamp

# Real Measurement
# True if this log comes from a real pulse measurement
# False if it comes from a timeout
bool status

# Velocity in odometer direction (m/s)
float32 vel
